lottie-swift/src/Public/Animation/AnimationView.swift

```diff
@@ -824,6 +820,11 @@ final public class AnimationView: LottieView {
   }
   
   override func animationMovedToWindow() {
+    /// Don't update any state if both the `superview` and `window` is `nil`
+    guard window != nil && superview != nil else {
+      return
+    }
```

Comment:
Please use early-return instead of new lines.

`guard window != nil && superview != nil else {
    return
  }`

Reply:
Hey @emrcftci, pushed another commit fixing the syntax :)